spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -20,20 +20,24 @@
 
 import numpy as np
 
+from spinalcordtoolbox.registration.algorithms import (register_step_label, register_step_ants_registration,
+                                                       register_step_ants_slice_regularized_registration,
+                                                       register_step_slicewise, register_step_slicewise_ants,
+                                                       Paramreg, ParamregMultiStep)
+from spinalcordtoolbox.registration.landmarks import register_landmarks
+
 from spinalcordtoolbox.metadata import get_file_label
 from spinalcordtoolbox.image import Image, add_suffix, generate_output_file, concat_warp2d
 from spinalcordtoolbox.centerline.core import ParamCenterline, get_centerline
 from spinalcordtoolbox.reports.qc import generate_qc
 from spinalcordtoolbox.resampling import resample_file
 from spinalcordtoolbox.math import dilate, binarize
-from spinalcordtoolbox.registration.algorithms import *
-from spinalcordtoolbox.registration.landmarks import *
 from spinalcordtoolbox.types import Coordinate
 from spinalcordtoolbox.utils.fs import (copy, extract_fname, check_file_exist, rmtree,
-                                        cache_save, cache_signature, cache_valid)
+                                        cache_save, cache_signature, cache_valid, tmp_create)
```

Comment:
I see, `tmp_create` is really from `utils.fs`, but it just happened to be imported through one of the `*` imports. (And the same for `run_proc` below.)

Reply:
Precisely!